docs/src/modules/components/ChartComponentsGrid.js

```diff
@@ -38,6 +38,12 @@ function components() {
       srcDark: '/static/x/component-illustrations/sparkline-dark.png',
       href: '/x/react-charts/sparkline/',
     },
+    // {
+    //   title: 'Gauge',
+    //   srcLight: '/static/x/component-illustrations/gauge-light.png',
+    //   srcDark: '/static/x/component-illustrations/gauge-dark.png',
+    //   href: '/x/react-charts/gauge/',
+    // },
```

Comment:
I will remove comments when the image is available

Reply:
Added illustrations for that in a separate PR → https://github.com/mui/mui-x/pull/12041